tests/io/dask/dataframe/test_update.py

```diff
@@ -171,6 +171,13 @@ def test_update_shuffle_buckets(
         range(unique_secondaries)
     )
 
+    assert set(dataset.table_meta["core"].names) == {
+        "primary",
+        "secondary",
+        "sorted_column",
+    }
+    assert _KTK_HASH_BUCKET not in dataset.table_meta["core"].names
```

Comment:
This is redundant given the previous statement but it's nice to have it explicit